about/index.html

```diff
@@ -1,12 +1,39 @@
-<html lang="eng">
+<!DOCTYPE html>
+<html lang="en">
 
-<head>
-  <meta charset="UTF-8">  
-  <meta name="viewport" content="width=device-width, initial scale=1">
-  <title>Welcome to my Paradox Collection!</title>
-</head>
+  <head>
+    <meta charset="UTF-8">  
+    <meta name="viewport" content="width=device-width, initial scale=1">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Each subpage should reference the same style.css from the homepage, if you need specific styling on a page you can add a new stylesheet overwriting some of that but keep `../style.css` loaded. And probably you can add custom, per subpage styles there too.

Reply:
It's really annoying that I can't answer your first comment directly with a general response :p  
Anyway, checkout the latest commit, I fixed everything you pointed out. Next I'll try and see if I can solve the index feature without using JS.

Reply:
Something, something, blame it on push force :P